packages/superset-ui-connection/test/SupersetClientClass.test.ts

```diff
@@ -125,6 +126,7 @@ describe('SupersetClientClass', () => {
     it('does not set csrfToken if response is not json', () => {
       fetchMock.get(LOGIN_GLOB, '123', {
         overwriteRoutes: true,
+        // @ts-ignore
```

Comment:
why does this one require a `@ts-ignore`?

Reply:
I see, I'm cool with `#3` if we also add a `@TODO` comment for what is needed to get rid of it.

otherwise `#1` seems next best / hopefully not many breaking changes, but understand that becomes a lot of yak-shaving 🤦‍♂️

Reply:
Ok, I stuck with #3 for now and added comments + an `@TODO`